packages/components/src/EditableText/EditableText.scss

```diff
@@ -69,25 +73,44 @@ $tc-input-editable-text-input-weight: $form-base-font-weight !default;
 		}
 	}
 	& &-form {
+		width: 100%;
 		display: flex;
 		align-items: center;
-		padding-left: $padding-large;
+		padding: {
+			left: $padding-large;
+			right: $padding-large;
+		}
+
+		:global(.form-group) {
+			width: 100%;
+			margin: 0;
+		}
 
 		& .tc-editable-text-form-input {
 			@include input-text($tc-input-editable-text-input-weight);
-			width: 18rem;
+			width: 100%;
+			padding-right: 2 * $tc-circle-editable-text-size + $padding-large;
 			&::selection {
 				background: lighten($scooter, 25%);
 			}
 		}
 
 		&-buttons {
 			display: flex;
+			position: absolute;
+			right: $padding-large;
+
+			display: flex;
```

Comment:
Why do you need them here?  I tested without them and it is ok.
```
align-items: center;
justify-content: center;
height: 32px;
top: 0;
```

Reply:
Without these props, buttons are not centered on IE.